package.json

```diff
@@ -14,19 +17,19 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/anthonykirby/lora-packet.git"
+    "url": "https://git.unidata.it/LORA/lora-packet"
```

Comment:
seriously?

Reply:
As i wrote it, it must not be merged. I needed it for validating project in my CI, but i forgot to remove.